src/main/java/org/taf/utils/RepositoryParser.java

```diff
@@ -0,0 +1,29 @@
+package org.taf.utils;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class RepositoryParser {
+    private final Properties propertyFile = new Properties();
+
+    private static final Logger LOGGER = LogManager.getLogger();
+
+    public RepositoryParser(String fileName) {
+
```

Comment:
minor: why empty line?

Reply:
What's wrong with empty line?

Reply:
Code should be consistent in composition approaches (if it suites coding principles) and style. Differently looking code complicates its perception.

Reply:
Thanks. I've found it (Code -> Reformat File. Check Rearrange code)